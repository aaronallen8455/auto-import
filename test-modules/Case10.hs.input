module Case10 where

main :: IO ()
main = f1
