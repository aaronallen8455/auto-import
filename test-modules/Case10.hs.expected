module Case10 where

import Foo (Foo(f1))

main :: IO ()
main = f1
